src/encoded/static/components/search.js

```diff
@@ -961,6 +975,11 @@ const SeriesComponent = ({ context: result, cartControls, removeConfirmation, au
     }
     if (assays.length > 0) {
         assays = assays.filter((item) => item !== 'pooled clone sequencing');
+        if (perturbationType?.length === 0) {
+            assaysTitle = _.uniq(assays);
+        } else if (perturbationType?.length > 0) {
+            assaysTitle = [...new Set(assaysTitle = assays?.map((a) => a.replace('CRISPR ', '')))];
+        }
         assays = _.uniq(assays);
```

Comment:
Do we still need this? The `Assays` data row that displays this seems nearly redundant now, displaying the same thing but without “CRISPR.”

Reply:
yeah we still do because L1055 still uses it. That's where the full assay titles with 'CRISPR' is displayed.